examples/applications/restapi/handlers.py

```diff
@@ -40,6 +40,7 @@ def add_routes(self):
             web.get(self.WALLETS_PARENT, self.get_parent_wallet),
             web.post(self.WALLETS_PARENT + "/load_wallet", self.load_wallet),
             web.get(self.WALLETS_ACCOUNT, self.get_account),
+            web.get(self.WALLETS_ACCOUNT + "/topup_account", self.topup_account),
```

Comment:
Is there any way to only expose these for regtest? Even if it's just a namespace for regtest specific methods, that errors if the network is not regtest.